source/common/common/assert.h

```diff
@@ -247,6 +247,8 @@ void resetEnvoyBugCountersForTest();
  */
 #define ENVOY_BUG(...) PASS_ON(PASS_ON(_ENVOY_BUG_VERBOSE)(__VA_ARGS__))
 
+#define IS_ENVOY_BUG(...) ENVOY_BUG(false, __VA_ARGS__);
```

Comment:
Could you please add a small comment on its usage ("Always triggering ENVOY_BUG intended for paths that are not expected to be reached.")?